atlasdb-workload-server-api/src/test/java/com/palantir/atlasdb/workload/transaction/witnessed/MaybeWitnessedTransactionTest.java

```diff
@@ -54,7 +54,7 @@ public void doesNotThrowWhenCommitTimestampPresentOnCreation() {
     public void toFullyWitnessedCopiesArgumentsCorrectly() {
         WitnessedWriteTransactionAction writeTransactionAction = mock(WitnessedWriteTransactionAction.class);
         WitnessedReadTransactionAction readTransactionAction = mock(WitnessedReadTransactionAction.class);
-        List<WitnessedTransactionAction> actions = List.of(readTransactionAction, writeTransactionAction);
+        List<WitnessedSingleCellTransactionAction> actions = List.of(readTransactionAction, writeTransactionAction);
```

Comment:
This appears unnecessary (at least the test runs fine without it, perhaps there's context later). But, in the interest of time, eh

Reply:
yeah this is supposed to be WTA, I think this was accidentally converted in refactoring